.github/labeler.yml

```diff
@@ -0,0 +1,6 @@
+'🔧 GitHub Configuration':
+  - '.github/**/*'
```

Comment:
This won't be possible until https://github.com/actions/labeler/issues/135 is resolved